### DISCUSSION TITLE
May I Join Contributors

### DISCUSSION THREAD

**Discussion by GarethMonty** [community/community#127598] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I would like to contribute to the repo. I have been using the sdk a lot and feel I can give back some.

**Comment by ebndev** [comment]
Hi @GarethMonty,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a specific. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

P.S would you mind telling us how you ended up reaching out for support in the Profile Category of Discussions? We are getting a lot of posts where folks are reaching out for support from projects hosted on GitHub and we want to ensure there aren't links or directions out there that may be misguiding folks.

**Reply by GarethMonty** [reply]
Hello, thank you for your response. I indeed posted this in the incorrect place...It was meant for the discussions area of a repo which I wanted to contribute to.
I believe I was in the discussion area of a repo, I was looking for a way to post, and I must have misinterpreted a link which lead here, and out of the context of the repo discussions.

Honestly it was late and not sure what I clicked to end up posting here...haha whoops! 

Again thank you checking in, nice to know people are always there to help!

**Reply by ebndev** [reply]
Thank you for your answer, it's very helpful! It has been happening to a lot of users so we will have to do some investigating 🧐
